app/javascript/components/lease_ups/leaseUpActions.js

```diff
@@ -5,13 +5,30 @@ import { performInSequence } from '~/utils/promiseUtils'
 
 export const getLeaseUpListings = async () => apiService.getLeaseUpListings()
 
-export const getApplications = async (listingId, page, filters) =>
-  apiService.fetchLeaseUpApplications(listingId, page, { filters }).then(({ records, pages }) => {
-    return {
-      records: map(records, buildLeaseUpAppPrefModel),
-      pages
-    }
-  })
+export const convertToCommaSeparatedList = (str) =>
+  // Take a series of phrases from the search box and turn it into a comma separated list.
+  str
+    ?.split(/ +/)
+    .filter((x) => x !== '')
+    .join(',')
+
+export const sanitizeAndFormatSearch = (str) => {
+  return convertToCommaSeparatedList(str.replace(/["']/g, ''))
```

Comment:
this strips out `" `and `'`s. Let me know if there's anything else you think we should strip out

Reply:
I don't know much about sanitizing inputs, so I did some googling and found a few opinions like this: https://kevinsmith.io/sanitize-your-inputs

Which suggest validation instead of sanitization, like our search wouldn't allow you to type quotes. The idea is that this way the user knows what's happening to their input, whereas sanitization changes the query behind the scenes.

Just something to think about, if we're already sanitizing on the ruby side using a built in function to protect against bad soql queries I think we should be good to leave it like this